app/src/main/res/values/string-not_translatable.xml

```diff
@@ -82,14 +82,11 @@ work. If not, see <https://creativecommons.org/publicdomain/zero/1.0/>.
     \n}
     \n</style>
     \n
-    \n
```

Comment:
the additional empty lines are there to faster see that these are different blocks. keep it.

as pf JS, keep it with 2 indent. I don't like 2space either, but on mobile you hardly see something in one line with spaces in front.